redirects.yaml

```diff
@@ -44,3 +44,46 @@ faq:
 mumble:
     en: https://docs.contao.org/manual/en/guides/mumble/
     de: https://docs.contao.org/manual/de/anleitungen/mumble/
+
+changelog/2.9:
+    en: https://github.com/contao/core/blob/2.9.5/CHANGELOG.txt
+changelog/2.10:
+    en: https://github.com/contao/core/blob/2.10.4/CHANGELOG.txt
+changelog/2.11:
+    en: https://github.com/contao/core/blob/2.11/CHANGELOG.md
+changelog/3.0:
+    en: https://github.com/contao/core/blob/3.0.6/system/docs/CHANGELOG.md
+changelog/3.1:
+    en: https://github.com/contao/core/blob/3.1.5/system/docs/CHANGELOG.md
+changelog/3.2:
+    en: https://github.com/contao/core/blob/3.2/system/docs/CHANGELOG.md
+changelog/3.3:
+    en: https://github.com/contao/core/blob/3.3.7/system/docs/CHANGELOG.md
+changelog/3.4:
+    en: https://github.com/contao/core/blob/3.4.5/system/docs/CHANGELOG.md
+changelog/3.5:
+    en: https://github.com/contao/core/blob/3.5/system/docs/CHANGELOG.md
+changelog/4.0:
+    en: https://github.com/contao/contao/blob/4.0.4/core-bundle/CHANGELOG.md
+changelog/4.1:
+    en: https://github.com/contao/contao/blob/4.1.3/core-bundle/CHANGELOG.md
+changelog/4.2:
+    en: https://github.com/contao/contao/blob/4.2.5/core-bundle/CHANGELOG.md
+changelog/4.3:
+    en: https://github.com/contao/contao/blob/4.3.11/core-bundle/CHANGELOG.md
+changelog/4.4:
+    en: https://github.com/contao/contao/blob/4.4/CHANGELOG.md
+changelog/4.5:
+    en: https://github.com/contao/contao/blob/4.5/CHANGELOG.md
+changelog/4.6:
+    en: https://github.com/contao/contao/blob/4.6/CHANGELOG.md
+changelog/4.7:
+    en: https://github.com/contao/contao/blob/4.7/CHANGELOG.md
+changelog/4.8:
+    en: https://github.com/contao/contao/blob/4.8/CHANGELOG.md
+changelog/4.9:
+    en: https://github.com/contao/contao/blob/4.9/CHANGELOG.md
+changelog/4.10:
+    en: https://github.com/contao/contao/blob/4.10/CHANGELOG.md
+changelog:
```

Comment:
Should be 4.10 because that's the current one very, very soon (https://github.com/contao/contao/pull/2161).